src/assets/scripts/aircraft/AircraftStripView.js

```diff
@@ -261,9 +261,9 @@ export default class AircraftStripView {
      * @param currentSpeedText {string}
      */
     updateAircraftTelemetryText(headingText, altitudeText, destinationText, currentSpeedText) {
-        this.$heading.text(headingText);
+        this.$heading.text(headingText.toUpperCase() );
```

Comment:
👍 

all these `.toUpperCase()` blocks, though somewhat annoying, are exactly where they belong now.  Our data should be stored in lowercase because it's easier to work with and is 'the natural state' of strings.  This is view logic, and as such, this is where the `toUpperCase` transformation should happen.